src/Prometheus/Internal/Pure/Base.hs

```diff
@@ -1,36 +1,47 @@
 {-# LANGUAGE DeriveFunctor
+           , DerivingStrategies
            , FlexibleInstances
            , FunctionalDependencies
  #-}
 
 {-# OPTIONS_HADDOCK hide #-}
-
-module Prometheus.Internal.Pure.Base where
-
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DeriveAnyClass #-}
```

Comment:
Should be formatted into the pragma block (can't commit a suggestion because Github sucks).

Reply:
Yep, I'll fix it.